aboutMe.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device=width,">
+    <link rel="stylesheet" href="main.css">
+    <script src="main.js" defer></script>
+    <ul class="nav">
+      <li><a href="index.html" class="link">Home</a></li>
+      <li><a href="aboutMe.html" class="link">About Me</a></li>
+      <li><a href="education.html" class="link">Education</a></li>
+      <li><a href="skills.html" class="link">Skills</a></li>
+      <li><a href="projects.html" class="link">Projects</a></li>
+      <li><a href="contact.html" class="link">Contact</a></li>
+    </ul>
+    </head>
+    <body>
+    <h1 class="intro">About Me:</h1>
+    <div class="flip-card-position">
+      <div class="flip-card">
+       <div class="flip-card-inner">
+      <div class="flip-card-front">
+    <img src="pursuit-Headshot.jpg" id="headshot2">
+    </div>
+    <div class="flip-card-back">
+      <img src="flag.jpg" id="headshot2">
+    </div>
+    </div>
+    </div>
+    </div>
+    <p id="intro-description">My name is Carlos Hidalgo and if I were to describe my true potential and motivation in 7 seconds it would be: <b>"Growing up in Puerto Rico, there weren't many technological opportunities so I made it my mission to bring the future of coding to them."</b> On top of that I also enjoy photography, video games, as well as helping around my community. </p>
+    <h1>Photography Work:</h1>
+    <button id="hidepics" onclick="hidePhoto()">Click Me</button>
```

Comment:
Use event listeners for these types of clicks in your JS instead of using inline function calls